src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -41,10 +44,57 @@ public void Initialize(ExtensionConfigContext context)
             {
                 throw new ArgumentNullException("context");
             }
+
+            INameResolver nameResolver = context.Config.NameResolver;
+            IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
```

Comment:
Fixed

---
In reply to: [63606377](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63606377) [](ancestors = 63606377,63605331)